inst/rmd/h/pandoc/elevate-section-attrs.js

```diff
@@ -0,0 +1,16 @@
+$(function() {
+  $("div[class*='section level']").each(function(i, el) {
+    var $el = $(el);
+    var headers = $el.children().filter(":header");
```

Comment:
All headers, or just the first child?

Reply:
Maybe it could just be the first...can you think of scenario where this matters @yihui?

Reply:
Each section has one and only one direct child, so it shouldn't matter. To be safe, we can select the first child.